src/components/dashboard/components/DashboardPage.jsx

```diff
@@ -0,0 +1,146 @@
+              Show Time : Sunday, 4 Feb 2024
+            </p>
+          </div>
+        </div>
+        <div className="card space-y-4">
+          <Image
+            alt="event image"
+            src={"http://via.placeholder.com/640x360"}
+            width={640}
+            height={360}
+          />
+          <div className="space-y-2">
+            <h2 className="text-md font-semibold">Lorem Ipsum</h2>
+            <p className="text-sm">
+              is simply dummy text of the printing and typesetting industry.
+              Lorem Ipsum has been the industry's standard dummy text ever since
+              the 1500s, when an unknown printer took a galley of type and
+              scrambled it to make a type specimen book.
+            </p>
+          </div>
+          <div>
+            <p className="text-sm text-gray-600">
+              Organized by : Mafatikhul Ilmi
+            </p>
+            <p className="text-sm text-gray-600">
+              Show Time : Sunday, 4 Feb 2024
+            </p>
+          </div>
+        </div>
+        <div className="card space-y-4">
+          <Image
+            alt="event image"
+            src={"http://via.placeholder.com/640x360"}
+            width={640}
+            height={360}
+          />
+          <div className="space-y-2">
+            <h2 className="text-md font-semibold">Lorem Ipsum</h2>
+            <p className="text-sm">
+              is simply dummy text of the printing and typesetting industry.
+              Lorem Ipsum has been the industry's standard dummy text ever since
+              the 1500s, when an unknown printer took a galley of type and
+              scrambled it to make a type specimen book.
+            </p>
+          </div>
+          <div>
+            <p className="text-sm text-gray-600">
+              Organized by : Mafatikhul Ilmi
+            </p>
+            <p className="text-sm text-gray-600">
+              Show Time : Sunday, 4 Feb 2024
+            </p>
+          </div>
+        </div>
+        <div className="card space-y-4">
+          <Image
+            alt="event image"
+            src={"http://via.placeholder.com/640x360"}
+            width={640}
+            height={360}
+          />
+          <div className="space-y-2">
+            <h2 className="text-md font-semibold">Lorem Ipsum</h2>
+            <p className="text-sm">
+              is simply dummy text of the printing and typesetting industry.
+              Lorem Ipsum has been the industry's standard dummy text ever since
+              the 1500s, when an unknown printer took a galley of type and
+              scrambled it to make a type specimen book.
+            </p>
+          </div>
+          <div>
+            <p className="text-sm text-gray-600">
+              Organized by : Mafatikhul Ilmi
+            </p>
+            <p className="text-sm text-gray-600">
+              Show Time : Sunday, 4 Feb 2024
+            </p>
+          </div>
```

Comment:
Great Job.

Please, Use array of objects and .map instead of hard coding the htmls.

Reply:
![image](https://github.com/alfonsusac/devscale-eventmaker/assets/5557896/57708bfb-4280-4e13-aafe-2b2d07e24c2b)